Ray.jl/src/Ray.jl

```diff
@@ -7,6 +7,7 @@ module Ray
 
 using ArgParse
 using Base64
+using CxxWrap: StdVector
```

Comment:
Not fully comfortable adding CxxWrap to the wrapper package but I need StdVector

Reply:
we may want to consider moving making an intenral `_submit_task` in the jll that the function here calls and does the C++ nonsense to convert the args into the form we need

Reply:
removed and punted the middleman function to the JLL